src/api/uniqueSdk/NFTController.ts

```diff
@@ -0,0 +1,95 @@
+import { INFTController } from '../chainApi/types';
+import { NFTCollection, NFTToken, TokenId } from '../chainApi/unique/types';
+import { Sdk } from '@unique-nft/sdk';
+import { CollectionInfo } from '@unique-nft/sdk/tokens';
+import { Settings } from '../restApi/settings/types';
+import { getEthAccount, normalizeAccountId } from './utils/addressUtils';
+import { checkTokenIsAllowed, filterAllowedTokens } from './utils/checkTokenIsAllowed';
+import { getTokenImage } from './utils/imageUtils';
+import { getAttributes } from './utils/attributesUtils';
+import config from '../../config';
+
+const { IPFSGateway } = config;
+
+export class UniqueSDKNFTController implements INFTController<NFTCollection, NFTToken> {
+  private sdk: Sdk;
+  private settings;
+  private allowedTokens: Record<number, string>;
+  constructor(sdk: Sdk, settings: Settings) {
+    this.sdk = sdk;
+    this.settings = settings;
+    this.allowedTokens = settings.blockchain?.unique?.allowedTokens.reduce((acc, item) => ({ ...acc, [item.collection]: item.tokens }), {}) || {};
+  }
+
+  async getAccountTokens(account: string): Promise<NFTToken[]> {
```

Comment:
:warning: 

getAccount**Marketable**Tokens



Let's rename it this way to avoid any confusion later on.